src/exchanges.js

```diff
@@ -0,0 +1,109 @@
+let Exchanges = require('pulse-publisher');
+
+/** Declaration of exchanges used by hooks  */
+let exchanges = new Exchanges({
+  title:       'Exchnages to manage hooks',
+  projectName: 'taskcluster-hooks',
+  serviceName: 'hooks',
+  version: 'v1',
+  description: [
+    'Hooks are responsible for creating tasks',
+    'at specific times or in response to webhooks and API calls.',
+    'Using this exchange allows us to make hooks which',
+    'repsond to particular pulse messages.',
+    '',
+    'For each of the exchanges defined below,',
+    'there is a listener which creates ,updates or deletes',
+    'a hook. Basically it creates another listener,',
+    'with the neccessary bindings, which creates the task',
+    'for every message it receives.',
```

Comment:
https://github.com/taskcluster/taskcluster-auth/blob/cdfc489c7fcb463d021ce6907786e58a450cbe78/src/exchanges.js#L11-L18 might provide a good template for this.

Reply:
Thanks!
Is the hooks service available at `hooks.taskcluster.net` ?

Reply:
It is!